### ISSUE TITLE
11.28 startup error

### ISSUE THREAD

**Issue by iantait** [ebaauw/homebridge-hue#485]
#### Issue

After removing homebridge, and installing it to update it, doing a npm install -g homebridge-hue homebridge-lib and starting homebridge (windows btw)
Anyone else seen this?

#### Log Messages

```
	[6/13/2019, 9:57:18 AM] Loaded plugin: homebridge-hue
	[6/13/2019, 9:57:18 AM] Registering platform 'homebridge-hue.Hue'
	[6/13/2019, 9:57:18 AM] ---
	[6/13/2019, 9:57:18 AM] Loaded plugin: homebridge-lib
	C:\Users\USER\AppData\Roaming\npm\node_modules\homebridge\lib\server.js:171
	      plugin.initializer(this._api);
	             ^
	
	TypeError: Class constructor homebridgeLib cannot be invoked without 'new'
	    at Server.<anonymous> (C:\Users\USER\AppData\Roaming\npm\node_modules\homebridge\lib\server.js:171:14)
	    at Array.forEach (<anonymous>)
	    at Server._loadPlugins (C:\Users\USER\AppData\Roaming\npm\node_modules\homebridge\lib\server.js:145:22)
	    at new Server (C:\Users\USER\AppData\Roaming\npm\node_modules\homebridge\lib\server.js:57:24)
	    at module.exports (C:\Users\USER\AppData\Roaming\npm\node_modules\homebridge\lib\cli.js:32:16)
	    at Object.<anonymous> (C:\Users\USER\AppData\Roaming\npm\node_modules\homebridge\bin\homebridge:17:22)
	    at Module._compile (internal/modules/cjs/loader.js:776:30)
	    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
	    at Module.load (internal/modules/cjs/loader.js:653:32)
	    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
```

**Comment by Lorccan** [comment]
I don’t know if you just left it out of your question, or whether this might be the problem, but the install should be:

`sudo npm -g i homebridge-lib homebridge-hue`

**Comment by iantait** [comment]
I had  npm install -g homebridge-hue homebridge-lib
I'll try it the other way around :-)

**Comment by Lorccan** [comment]
Not sure if it makes a difference, but worth a go. I know there’s a dependency on the lib, so it makes sense that it would go first.

**Comment by iantait** [comment]
Didn't seem to make a difference - same error.

**Comment by ebaauw** [comment]
Most likely you still have an old copy of homebridge-lib and/or homebridge-hue somewhere on your system.  I have no experience running homebridge Windows, but I don’t think `npm -g` installs stuff to the user’s APPDATA.

**Comment by ebaauw** [comment]
Fixed in homebridge-lib@4.2.8, see https://github.com/ebaauw/homebridge-lib/issues/11.

**Comment by iantait** [comment]
Just installed it - all good.

Thank you!
